package exceptionhandling;

public class ArrayDemo {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try
		{
			
		
		int a[]= {2,3,4};
		System.out.println(a[7]);
		}
		catch(Exception e)
		{
			System.out.println(e);
		}
		

	}

}
